frontend/src/BudgetModeDisplay.tsx

```diff
@@ -20,23 +25,40 @@ export function BudgetModeDisplay({
     currentFleetSupply,
     maxFleetSupply,
     unitCounts,
+    isFighterUpgraded,
+    shipCapacityUsed,
+    maxShipCapacity,
+    spaceDockFighterBonus,
 }: Props) {
+    const shipCapacityRemaining =
```

Comment:
:pick: This variable name kinda bothers me, only because elsewhere you're careful to distinguish between "ship capacity" and the "space dock fighter bonus" (I'm guessing to later account for ground forces, which can't be supported by space docks?). But here it's all lumped together and called "ship capacity".

A few alternate proposals:
`totalCapacityRemaining`
`capacityRemaining`
`fighterCapacityRemaining` (I think maybe I dislike this one the least)

Reply:
Right guess on the future ground forces problem, and yes good point - I like `fighterCapacityRemaining` since that's the least ambiguous, and can handle renaming it when we add advanced mode!